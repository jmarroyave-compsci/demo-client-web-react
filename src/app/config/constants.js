import _package_ from 'package.json';

const Constants = {
	DEBUG : process.env.NEXT_PUBLIC__DEBUG,
	LOCAL : process.env.NEXT_PUBLIC__LOCAL,
	APP : {
		BASE_PATH: process.env.NEXT_PUBLIC__SITE_URL,
		SITE_URL: process.env.NEXT_PUBLIC__SITE_URL,
		VERSION : _package_.version,
		TITLE: process.env.NEXT_PUBLIC__TITLE,
		PAGE_TITLE: process.env.NEXT_PUBLIC__PAGE_TITLE
	},
	SERVERS: {
		DATA: process.env.NEXT_PUBLIC__DATA, 
		GRAPHQL :  process.env.NEXT_PUBLIC__DATA_GRAPHQL,
	},
	DEFAULTS : {
		PAGING_PAGE_SIZE: 10,
	},
	PLUGINS : {
		GOOGLE: {
			ANALYTICS: process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS,
			MAPS: process.env.NEXT_PUBLIC__GOOGLE_MAPS_KEY,
		},
	},
	FEATURES : {
		SEARCH : (process.env.NEXT_PUBLIC__FEATURE_SEARCH == "true"),
	},
}

export default Constants;