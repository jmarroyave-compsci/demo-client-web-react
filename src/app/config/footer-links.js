import _links from './links';

const links = [
  { "name" : "jmarroyave", items: [
    { "name": "blog", "link" : _links.blog },
  ] },
  { "name" : "labs", items: [
    { "name": "stackoverflow", "link" : _links.labs_trends },
    { "name": "mlb", "link" : _links.labs_mlb },
    { "name": "search", "link" : _links.labs_search },
  ] },
  { "name" : "knowledge base", items: [
    { "name": "android", "link" : _links.kb_android },
  ] },
];

export default links;