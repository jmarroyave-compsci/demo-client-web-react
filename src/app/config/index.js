import _package_ from 'package.json';

const Constants = {
	TEST: (process.env.TESTING === "true") ? true : false,
	DEBUG : process.env.NEXT_PUBLIC__DEBUG,
	LOCAL : process.env.NEXT_PUBLIC__LOCAL,
	APP : {
		BASE_PATH: process.env.NEXT_PUBLIC__BASE_PATH,
		SITE_URL: process.env.NEXT_PUBLIC__SITE_URL,
		VERSION : _package_.version,
		TITLE: process.env.NEXT_PUBLIC__TITLE,
		DESCRIPTION: process.env.NEXT_PUBLIC__DESCRIPTION,
		PAGE_TITLE: process.env.NEXT_PUBLIC__PAGE_TITLE,
		HOME_URL: `${process.env.NEXT_PUBLIC__SITE_URL}${process.env.NEXT_PUBLIC__BASE_PATH}`,
	},
	SERVERS: {
		DATA: process.env.NEXT_PUBLIC__SERVER, 
		WEB_SOCKETS: process.env.NEXT_PUBLIC__SERVER, 
		GRAPHQL :  `${process.env.NEXT_PUBLIC__SERVER}/${process.env.NEXT_PUBLIC__SERVER_API_VERSION}/graphql`,
	},
	DEFAULTS : {
		PAGING_PAGE_SIZE: 10,
		IMAGE_PLACEHOLDER: "/img/ph.jpg",
	},
	PLUGINS : {
		GOOGLE: {
			ANALYTICS: process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS,
			MAPS: process.env.NEXT_PUBLIC__GOOGLE_MAPS_KEY,
		},
	},
	FEATURES : {
		SEARCH : (process.env.NEXT_PUBLIC__FEATURE_SEARCH == "true"),
	},
	SOCIAL_NETWORKS: {
		CARD : {
			TITLE : `${process.env.NEXT_PUBLIC__TITLE} v.${_package_.version}`,
			THUMBNAIL_TWITTER: `${process.env.NEXT_PUBLIC__BASE_PATH}${process.env.NEXT_PUBLIC__CARD_THUMBNAIL_TWITTER}`,
		},
		TWITTER: process.env.NEXT_PUBLIC__SOCIAL_NETWORKS_TWITTER,
	}
}

export default Constants;