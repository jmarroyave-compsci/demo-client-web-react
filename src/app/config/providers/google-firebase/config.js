const config = {
  apiKey: "AIzaSyDaNheMMUtsqx8AjiJAWDcfXwAEBYLJUVY",
  authDomain: "labs-f885b.firebaseapp.com",
  projectId: "labs-f885b",
  //projectId: "labs-f885",
  storageBucket: "labs-f885b.appspot.com",
  messagingSenderId: "606766444808",
  appId: "1:606766444808:web:41eec12c129d582727bcbd",
  measurementId: "G-TNBYCW57TS"
};

export default config;