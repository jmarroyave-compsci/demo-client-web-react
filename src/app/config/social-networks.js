const links = [
  /*
	{enable: true, url: 'https://plus.google.com/112618720945759819017', icon: 'google-plus', color: '#d34836'},
	{enable: true, url: 'https://twitter.com/jmarroyave', icon: 'twitter', color: '#00aced'},
	*/
  { enable: true, url: 'https://facebook.com/jmarroyave', icon: 'facebook', color: '#3b5998' },
  { enable: true, url: 'https://www.linkedin.com/in/jmarroyave', icon: 'linkedin', color: '#0077B5' },
  { enable: true, url: 'https://github.com/jmarroyave', icon: 'github', color: '#000' },
  { enable: true, url: 'mailto:jmarroyave.compsci@gmail.com', icon: 'envelope-o', color: '#000' },
];


export default links;