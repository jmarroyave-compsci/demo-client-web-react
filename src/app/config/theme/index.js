import React from 'react';
import { createTheme } from '@mui/material/styles';
import { red } from '@mui/material/colors';

// Create a theme instance.
export const getTheme = ( mode ) => {
  return createTheme({
    palette: {
      mode: mode,
      primary: {
        main: '#556cd6',
      },
      secondary: {
        main: '#19857b',
      },
      error: {
        main: red.A400,
      },
    },
  });
}