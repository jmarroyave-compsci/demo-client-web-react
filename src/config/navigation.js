
class Navigation {
  constructor(){
    this.drawer = {
      show: false,
    };

    this.options = [
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'Home', 
        icon: 'home', 
        route: '/', 
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        separator: true,
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'Movies', 
        icon: 'movie', 
        route: '/movies', 
      },
        {
          showInMenu: false, 
          showInDrawer: false, 
          name: 'Movie', 
          icon: 'movie', 
          route: '/movie/:id/:name', 
        },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'Podcasts', 
        icon: 'podcasts', 
        route: '/podcasts', 
      },
        {
          showInMenu: false, 
          showInDrawer: false, 
          name: 'Podcast', 
          icon: 'podcast', 
          route: '/podcast/:id/:name', 
        },
      {
        showInMenu: true, 
        showInDrawer: true, 
        separator: true,
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'People', 
        icon: 'people', 
        route: '/people', 
      },
        {
          showInMenu: false, 
          showInDrawer: false, 
          name: 'Person', 
          icon: 'person', 
          route: '/person/:id/:name', 
        },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'Awards', 
        route: '/stories/awards/:entity?/:year?/:page?', 
      },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'Stories', 
        route: '/stories', 
      },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'PodcastMusic', 
        route: '/podcasts/stories/music/:page?', 
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        separator: true,
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'About', 
        icon: 'info', 
        route: '/about', 
      },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'Search', 
        icon: 'search', 
        route: '/search/:qry/:page?', 
      },
    ];
  }
}



const nav = new Navigation();
export default nav;