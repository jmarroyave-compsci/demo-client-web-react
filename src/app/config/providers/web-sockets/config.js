import config from 'app/config';

const CONF = {
  SERVER: config.SERVERS.WEB_SOCKETS
};

export default CONF;