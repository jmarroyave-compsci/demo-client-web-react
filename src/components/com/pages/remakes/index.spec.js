import React from 'react';
import {render, fireEvent, waitFor, screen} from '@testing-library/react'
import '@testing-library/jest-dom'

import { itMustHaveNoErrors, TestProvider } from "tests/jest/shared"

import itemState from './automata/tests/item.json'
import TestComponent from '.';
import config from './.config';
 
describe(`${config.automata.name.toUpperCase()}`, () => {
  let store;
  let component;
 
  beforeEach(() => { 
  });
 
 
  it('compare against snapshot', () => {
    const renderer = render(
      <TestProvider state={{[config.automata.name]: itemState}}>
        <TestComponent render="list"/>
      </TestProvider>
    )

    itMustHaveNoErrors(renderer)
    expect(renderer.container).toMatchSnapshot();
  });
});
