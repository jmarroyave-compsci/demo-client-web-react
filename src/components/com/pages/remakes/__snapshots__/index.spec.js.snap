// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`STORIES_REMAKES compare against snapshot 1`] = `
@keyframes animation-0 {
  0% {
    opacity: 1;
  }

  50% {
    opacity: 0.4;
  }

  100% {
    opacity: 1;
  }
}

@keyframes animation-0 {
  0% {
    opacity: 1;
  }

  50% {
    opacity: 0.4;
  }

  100% {
    opacity: 1;
  }
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-1 {
  box-sizing: border-box;
  width: 100%;
  height: inherit;
  border-radius: 0;
  background: linear-gradient(60deg, #1976d2 75%, #000 150%);
  color: #fff;
}

@media (min-width:0px) {
  .emotion-1 {
    padding: 1.5rem 1.5rem 1.5rem 1.5rem;
  }
}

@media (min-width:600px) {
  .emotion-1 {
    padding: 2rem 3rem 2rem 3rem;
  }
}

@media (min-width:900px) {
  .emotion-1 {
    padding: 3rem 2.5rem 3rem 2.5rem;
  }
}

.emotion-3 {
  padding: 0 0 1rem 0;
  margin: 0;
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  font-size: 1rem;
}

@media (min-width:0px) {
  .emotion-4 {
    padding: 0rem 2rem 3rem 2rem;
  }
}

@media (min-width:600px) {
  .emotion-4 {
    padding: 0rem 2rem 3rem 2rem;
  }
}

@media (min-width:1200px) {
  .emotion-4 {
    padding: 0rem 4rem 3rem 4rem;
  }
}

.emotion-5 {
  padding: 0rem 0rem 4rem 0rem;
}

.emotion-6 {
  margin: 0 0 2rem 0;
  padding: 0;
  width: 100%;
}

.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-8>:not(style)+:not(style) {
  margin: 0;
  margin-top: 16px;
}

.emotion-9 {
  margin: 0;
  padding: 0 1rem 1rem 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  overflow: hidden;
  width: 100%;
  color: rgba(0, 0, 0, 0.87);
  background-color: transparent;
}

.emotion-10 {
  background-color: #fff;
  color: rgba(0, 0, 0, 0.87);
  -webkit-transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border-radius: 4px;
  box-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  margin: 0.1rem;
  width: 100%;
}

.emotion-11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  width: 100%;
}

.emotion-12 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-12>:not(style)+:not(style) {
  margin: 0;
  margin-top: 8px;
}

.emotion-13 {
  display: block;
  background-color: rgba(0, 0, 0, 0.11);
  height: auto;
  margin-top: 0;
  margin-bottom: 0;
  transform-origin: 0 55%;
  -webkit-transform: scale(1, 0.60);
  -moz-transform: scale(1, 0.60);
  -ms-transform: scale(1, 0.60);
  transform: scale(1, 0.60);
  border-radius: 4px/6.7px;
  -webkit-animation: animation-0 1.5s ease-in-out 0.5s infinite;
  animation: animation-0 1.5s ease-in-out 0.5s infinite;
}

.emotion-13:empty:before {
  content: "\\00a0";
}

.emotion-14 {
  display: block;
  background-color: rgba(0, 0, 0, 0.11);
  height: 1.2em;
  -webkit-animation: animation-0 1.5s ease-in-out 0.5s infinite;
  animation: animation-0 1.5s ease-in-out 0.5s infinite;
}

.emotion-23 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
}

.emotion-23>:not(style)+:not(style) {
  margin: 0;
  margin-left: 16px;
}

.emotion-24 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.875rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 5px 15px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border: 1px solid rgba(25, 118, 210, 0.5);
  color: #1976d2;
}

.emotion-24::-moz-focus-inner {
  border-style: none;
}

.emotion-24.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-24 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-24:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: rgba(25, 118, 210, 0.04);
  border: 1px solid #1976d2;
}

@media (hover: none) {
  .emotion-24:hover {
    background-color: transparent;
  }
}

.emotion-24.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
  border: 1px solid rgba(0, 0, 0, 0.12);
}

.emotion-26 {
  background-color: #fff;
  color: rgba(0, 0, 0, 0.87);
  -webkit-transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border-radius: 4px;
  box-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);
}

.emotion-27 {
  color: rgba(0, 0, 0, 0.87);
  background-color: #fff;
  padding: 1rem;
}

.emotion-28 {
  font-size: 1.5rem;
  padding-bottom: 0.5rem;
  color: inherit;
}

.emotion-29 {
  font-size: 0.9rem;
  color: inherit;
}

.emotion-30 {
  cursor: pointer;
  color: inherit;
  background-color: transparent;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  text-decoration-style: dotted;
  text-decoration-color: inherit;
  text-decoration-thickness: 1px;
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
  display: inline-block;
  width: 100%;
}

<div>
  <div
    class="emotion-0"
  >
    <div
      style="margin-bottom: 2rem;"
    >
      <div
        class="com-layout-section section-0 MuiBox-root emotion-1"
      >
        <div
          class="emotion-0"
        >
          <h2
            class="emotion-3"
          >
            Remakes
          </h2>
          
        </div>
      </div>
    </div>
    <div
      class="emotion-4"
    >
      <div
        class="MuiGrid-root page-module MuiGrid-container"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-9"
        >
          <div
            class="emotion-5"
          >
            <div
              class="emotion-6"
            >
              <div />
              <div
                class="emotion-6"
              >
                <div
                  class="emotion-8"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-content-xs-center"
                    style="width: 100%;"
                  >
                    <div
                      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
                    >
                      <div
                        class="emotion-9"
                      >
                        <div
                          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 emotion-10"
                        >
                          <div
                            class="emotion-11"
                          >
                            <div
                              class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
                              style="background-color: transparent; flex-grow: 1; padding: 0.5rem;"
                            >
                              <div
                                class="emotion-12"
                              >
                                <span
                                  class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse emotion-13"
                                  style="width: 80%;"
                                />
                                <span
                                  class="MuiSkeleton-root MuiSkeleton-rectangular MuiSkeleton-pulse emotion-14"
                                  style="height: 240px;"
                                />
                                <span
                                  class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse emotion-13"
                                />
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
                    >
                      <div
                        class="emotion-9"
                      >
                        <div
                          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 emotion-10"
                        >
                          <div
                            class="emotion-11"
                          >
                            <div
                              class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
                              style="background-color: transparent; flex-grow: 1; padding: 0.5rem;"
                            >
                              <div
                                class="emotion-12"
                              >
                                <span
                                  class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse emotion-13"
                                  style="width: 80%;"
                                />
                                <span
                                  class="MuiSkeleton-root MuiSkeleton-rectangular MuiSkeleton-pulse emotion-14"
                                  style="height: 240px;"
                                />
                                <span
                                  class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse emotion-13"
                                />
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="emotion-23"
                  >
                    <button
                      class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root Mui-disabled emotion-24"
                      disabled=""
                      tabindex="-1"
                      type="button"
                    >
                      Back
                    </button>
                    <button
                      class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root Mui-disabled emotion-24"
                      disabled=""
                      tabindex="-1"
                      type="button"
                    >
                      Next
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-3"
        >
          <div
            style="padding-left: 1rem; padding-right: 0.5rem;"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 emotion-26"
            >
              <div
                class="emotion-27"
              >
                <div
                  class="emotion-28"
                >
                  See also
                </div>
                <div
                  class="emotion-29"
                >
                  <span
                    class="emotion-30"
                  >
                    Movie Awards
                  </span>
                </div>
                <div
                  class="emotion-29"
                >
                  <span
                    class="emotion-30"
                  >
                    Movie Remakes
                  </span>
                </div>
                <div
                  class="emotion-29"
                >
                  <span
                    class="emotion-30"
                  >
                    Movie Directors
                  </span>
                </div>
                <div
                  class="emotion-29"
                >
                  <span
                    class="emotion-30"
                  >
                    Movie Producers
                  </span>
                </div>
                <div
                  class="emotion-29"
                >
                  <span
                    class="emotion-30"
                  >
                    Movie Actors
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
